clients/cli/src/debug.rs

```diff
@@ -0,0 +1,108 @@
+use lockbook_core::Core;
+use lockbook_core::{FileMetaMapExt, FileMetaVecExt};
+use lockbook_core::{TestRepoError, Warning};
+
+use crate::error::CliError;
+use crate::selector::select_meta;
+use crate::Debug::*;
```

Comment:
I'd move this to the first line of the `debug` fn below.

Reply:
7d1ac448